client/packages/system/src/Patient/ListView/ListView.tsx

```diff
@@ -74,6 +74,14 @@ const PatientListComponent: FC = () => {
   const columnDefinitions: ColumnDescription<PatientRowFragment>[] = [
     { key: 'code', label: 'label.patient-id' },
     { key: 'code2', label: 'label.patient-nuic' },
+    {
+      key: 'createdDatetime',
+      label: 'label.created',
+      width: 175,
```

Comment:
You can reduce the width here to 110. However, I also tried to reduce the width of the dob column which works with 150 which also would work for the createdDatetime and would give a consistent column width. Another option is to remove the width option for both columns, which works fine for me and maybe makes the best use of the page width?